_posts/0000-01-02-azaimirul.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
Last but not least, thank you for anyone who read all these comments from the beginning till the end. I hope that we can have a better life in the future. Thank You.